Jenkinsfile

```diff
@@ -0,0 +1,143 @@
+pipeline {
+  agent {
+    label "xenial-intel-device-plugins"
+  }
+  options {
+    timeout(time: 2, unit: "HOURS")
+  }
+  environment {
+    RUNC_VERSION="v1.0.0-rc6"
+    CRIO_VERSION="v1.11.5"
+    OSTREE_VERSION="v2018.8"
+    BUILDAH_VERSION="v1.4"
+    GO_VERSION="1.11.11"
+    GO_TAR="go${GO_VERSION}.linux-amd64.tar.gz"
+    GOROOT="/usr/local/go"
+    GOPATH="/tmp/go"
+    PATH="${env.PATH}:/usr/local/go/bin:${GOPATH}/bin"
+    REPO_NAME="intel-device-plugins-for-kubernetes"
+    REPO_DIR="$GOPATH/src/github.com/intel/${REPO_NAME}"
+  }
+  stages {
+    stage("Get requirements") {
+      steps {
+        sh "curl -O https://dl.google.com/go/${GO_TAR}"
+        sh "tar -xvf $GO_TAR"
+        sh "sudo mv go $GOROOT"
+        sh "sudo apt-get -y install e2fslibs-dev libfuse-dev libgpgme11-dev libdevmapper-dev libglib2.0-dev libprotobuf-dev"
```

Comment:
Are we protecting jenkins sudo somehow?

Reply:
No for this kind of worker, Jenkinsfile runs on a disposable vm, as soon as the job ends, the virtual machine is deleted